ingestion_tools/scripts/importers/alignment.py

```diff
@@ -0,0 +1,145 @@
+        self.converter = alignment_converter_factory(config, metadata, path, parents)
+
+    def import_metadata(self) -> None:
+        metadata_path = self.get_metadata_path()
+        if metadata_path in self.written_metadata_files:
+            print(f"Skipping rewriting metadata for alignment {self.path}")
+            return
+        try:
+            meta = AlignmentMetadata(self.config.fs, self.get_deposition().name, self.get_base_metadata())
+            meta.write_metadata(metadata_path, self.get_extra_metadata())
+            self.written_metadata_files.add(metadata_path)
+        except IOError:
+            print("Skipping creating metadata for default alignment with no source tomogram")
+
+    def import_item(self) -> None:
+        if self.is_default_alignment() or not self.is_valid():
+            print(
+                f"Skipping importing alignment with path {self.path} as it is either a default alignment or doesn't"
+                " have dimension data",
+            )
+            return
+        super().import_item()
+
+    def get_output_path(self) -> str:
+        output_directory = super().get_output_path()
+        return os.path.join(output_directory, f"{self.identifier}-")
+
+    def get_dest_filename(self) -> str | None:
+        if not self.path:
+            return None
+        output_dir = self.get_output_path()
+        return f"{output_dir}{os.path.basename(self.path)}"
+
+    def get_metadata_path(self) -> str:
+        return self.get_output_path() + "alignment_metadata.json"
+
+    def get_extra_metadata(self) -> dict:
+        extra_metadata = {
+            "per_section_alignment_parameters": self.converter.get_per_section_alignment_parameters(),
+            "alignment_path": self.converter.get_alignment_path(),
+            "tilt_path": self.converter.get_tilt_path(),
+            "tiltx_path": self.converter.get_tiltx_path(),
+        }
+        if "volume_dimension" not in self.metadata:
+            extra_metadata["volume_dimension"] = self.get_tomogram_volume_dimension()
+        return extra_metadata
+
+    def get_tomogram_volume_dimension(self) -> dict:
+        for tomogram in TomogramImporter.finder(self.config, **self.parents):
+            return tomogram.get_source_volume_info().get_dimensions()
+
+        # If no source tomogram is found don't create a default alignment metadata file.
+        raise IOError("No source tomogram found for creating default alignment")
+
+    def is_default_alignment(self) -> bool:
+        return self.name.lower() == "default"
+
+    def is_valid(self) -> bool:
+        volume_dim = self.metadata.get("volume_dimension", {})
+        return all(volume_dim.get(dim) for dim in "xyz") or next(
+            TomogramImporter.finder(self.config, **self.parents),
+            None,
+        )
+
+    @classmethod
+    def get_default_config(cls) -> list[dict] | None:
+        return [
+            {
```

Comment:
should these defaults align with the defaults [here](https://github.com/chanzuckerberg/cryoet-data-portal/issues/997)?